From a8475278880270a3bf682d9552e26651081330ce Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Fri, 11 Aug 2017 21:19:28 +0200
Subject: [PATCH] tests: Label parts with decimals, not roman numerals

dblatex (via gtkdoc-mkpdf) warns that labels must be integers:

    Warning: only an integer in @label can be processed: 'I'
    Warning: only an integer in @label can be processed: 'II'
    Warning: only an integer in @label can be processed: 'III'

Make the labels decimal to placate dblatex. I'm not sure this is
actually the best solution, though.

https://bugzilla.gnome.org/show_bug.cgi?id=786174
---
 tests/annotations/docs/tester-docs.xml | 4 ++--
 tests/gobject/docs/tester-docs.xml     | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/tests/annotations/docs/tester-docs.xml b/tests/annotations/docs/tester-docs.xml
index e2abf74..7d80f9d 100644
--- a/tests/annotations/docs/tester-docs.xml
+++ b/tests/annotations/docs/tester-docs.xml
@@ -16,7 +16,7 @@
     </releaseinfo>
   </bookinfo>
 
-  <reference label="II">
+  <reference label="2">
     <title>API Reference</title>
     <chapter id="main-api">
       <title>Tests</title>
@@ -24,7 +24,7 @@
     </chapter>
   </reference>
 
-  <part label="III">
+  <part label="3">
     <title>Appendix</title>
 
     <index id="api-index">
diff --git a/tests/gobject/docs/tester-docs.xml b/tests/gobject/docs/tester-docs.xml
index a534c55..05117ef 100644
--- a/tests/gobject/docs/tester-docs.xml
+++ b/tests/gobject/docs/tester-docs.xml
@@ -37,7 +37,7 @@
     </para>
   </preface>
   
-  <part label="I" id="part.i">
+  <part label="1" id="part.i">
     <title>Overview</title>
     <chapter id="Overview-building">
       <title>How to build the library</title>
@@ -45,7 +45,7 @@
     </chapter>
   </part>
 
-  <reference label="II" id="part.ii">
+  <reference label="2" id="part.ii">
     <title>API Reference</title>
     <chapter id="main-api">
       <title>Tests</title>
@@ -58,7 +58,7 @@
     </chapter>
   </reference>
 
-  <part label="III" id="part.iii">
+  <part label="3" id="part.iii">
     <title>Appendix</title>
 
     <chapter id="object-tree">
-- 
2.13.0

